qemu/tests/block_copy.py

```diff
@@ -456,3 +456,17 @@ def reopen(self, reopen_image):
         self.params["image_name_%s" % self.tag] = reopen_image
         self.vm.create(params=self.params)
         self.vm.verify_alive()
+
+    def create_files(self):
+        """
+        Create files and record m5 values of them.
+        """
+        file_names = self.params["file_names"]
+        map(self.create_file, file_names.split())
+
+    def verify_md5s(self):
+        """
+        Check if the md5 values matches the record ones.
+        """
+        file_names = self.params["file_names"]
+        map(self.verify_md5, file_names.split())
```

Comment:
This party also appears in https://github.com/autotest/tp-qemu/pull/1017
You may delete one depends on which patch merges first.

Reply:
@hereischen Yes, just keep here for testing pass, I will delete the redundant part once one of them got merged. Thanks.